dashboard/urls.py

```diff
@@ -18,24 +18,27 @@
 from django.contrib.auth.decorators import login_required
 
 from django.conf import settings
-from django.conf.urls import include, url
+from django.conf.urls import include
 from django.conf.urls.static import static
 from django.urls import path, re_path
 
 from dashboard.graphql.view import DashboardGraphQLView
 
 from django.views.decorators.cache import cache_page
 
-from . import views
+from . import views, ltiv1p3
```

Comment:
Again, I'm concerned about naming our module for a specific version of the standard, `ltiv1p3`.